### PR TITLE
Remove UGH (4)

### PR DESCRIPTION
**Pull request by matthias-ronge**
Removes all remaining references to UGH interface methods

[compare against part 3](https://github.com/matthias-ronge/kitodo-production/compare/KIT-1155.3...KIT-1155.4)

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/production/forms/ProzesskopieForm.java [hunk 1]
```diff
@@ -531,32 +524,7 @@ private DocStructInterface getDocStruct(AdditionalField field) throws Preference
...
     }
 
-    /**
-     * Get together authors' names.
-     *
-     * @param persons
-     *            list of persons
-     * @return authors' names as String
-     */
-    protected String getAuthors(List<PersonInterface> persons) {
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/production/forms/ProzesskopieForm.java]: why did you remove this?

Comment (matthias-ronge) [reply Kitodo/src/main/java/org/kitodo/production/forms/ProzesskopieForm.java]: Because there is no dedicated `Person` object any more in the future. Complex object descriptions, such as persons, can and have to be done using meta-data groups.
